analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -77,9 +69,33 @@ class Gradle(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConfig
                 Gradle(analyzerConfig, repoConfig)
     }
 
-    val maven = MavenSupport { localRepositoryManager ->
-        GradleLocalRepositoryManager(localRepositoryManager)
-    }
+    val maven = MavenSupport(object : WorkspaceReader {
+        private val workspaceRepository = WorkspaceRepository()
```

Comment:
Consider passing "gradleCache" as type instead of using the default "workspace", because this reader is not actually reading the workspace, but the Gradle cache.

Reply:
Done.